packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -399,7 +419,9 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
 
     this.__focusInside = false;
 
-    if (!this.__hoverInside) {
+    // When both tooltip and its focused target are detached,
+    // avoid closing twice: both on detach and target blur.
+    if (!this.__hoverInside && this._autoOpened) {
```

Comment:
Not sure what would be the correct way to test this, as I'd like to avoid using spies for internal methods.

Reply:
Removed this check for now as it breaks test related to closing while not yet fully opened.